src/IPC/MasterIPC.ts

```diff
@@ -2,10 +2,10 @@ import { EventEmitter } from 'events';
 import { Server, NodeMessage } from 'veza';
 import { Util } from 'discord.js';
 import { ShardingManager } from '..';
-import { isMaster } from 'cluster';
+import cluster from 'cluster';
 import { IPCEvents, SharderEvents } from '../Util/Constants';
 import { IPCRequest } from './ClusterIPC';
-
+const isPrimary = cluster.isPrimary
```

Comment:
please import isPrimary from cluster directly in the above import statement, no extra const needed